dask/dataframe/io/parquet/core.py

```diff
@@ -1394,6 +1396,8 @@ def apply_conjunction(parts, statistics, conjunction):
                         and max >= value
                         or operator == "in"
                         and any(min <= item <= max for item in value)
+                        or operator == "not in"
+                        and not any(min == max == item for item in value)
```

Comment:
Should we raise on unhandled operators instead of silently ignoring them?

Reply:
Ah, yes. I agree that the problem is that we were silently filtering for an "unsupported" operator.  I'll try to make the list of supported operators more explicit.

Reply:
:tada: